package.json

```diff
@@ -61,16 +62,20 @@
     "eslint-plugin-unicorn": "^47.0.0",
     "fast-json-stable-stringify": "~2.1.0",
     "fdir": "~6.0.1",
+    "fs-extra": "11.1.1",
     "husky": "^8.0.1",
     "json-schema-to-typescript": "~10.1.5",
+    "klaw": "4.1.0",
```

Comment:
Can you use fdir instead? That's an existing dependency.

Reply:
Its now using fdir with `withFullPaths`.